reactivexcomponent/reactivexcomponent/configuration/api_configuration.py

```diff
@@ -0,0 +1,96 @@
+""" API Configuartion """
+
+from lxml import etree
+
+EVENT_TYPE = {"Update": "UPDATE", "Error": "ERROR"}
```

Comment:
Looks like you're trying to define an enumeration using dictionaries. If it's the case change it to separate constants.